styles/variables-carbon.less

```diff
@@ -319,18 +319,29 @@
 // ---------------------------------------
 @agate-spinner-size: 52px;
 @agate-spinner-bg-padding: 9px;
+@agate-spinner-huge-size: (@agate-spinner-size * 2.5);
+@agate-spinner-huge-bg-padding: (@agate-spinner-bg-padding * 2.5);
 @agate-spinner-small-size: 36px;
 @agate-spinner-small-bg-padding: 5px;
+@agate-spinner-smallest-size: 24px;
 @agate-spinner-searching-width: (@agate-spinner-size / 13);
```

Comment:
I'm still confused by the `/ 13` value here and the `* 3` below. Perhaps this should just be set to `4px` and the below value set to `12px`? It seems like the fixed values would be easier to mentally process and update (if needed) rather than fifty-two thirteeths of the width.